substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/resources/NativeImageResourceFileSystem.java

```diff
@@ -0,0 +1,1206 @@
+package com.oracle.svm.core.jdk.resources;
```

Comment:
Copyright header is missing in all new files.

Reply:
Done.